gradle/publish-dists.gradle

```diff
@@ -0,0 +1,64 @@
+apply plugin: 'maven-publish'
+apply plugin: 'com.jfrog.artifactory'
+apply plugin: 'com.jfrog.bintray'
+
+// If you modify bintray or artifactory publishing you also need to modify publish-jars.gradle
+
+publishing {
+  publications {
+    dists(MavenPublication) {
+      artifact distTar
```

Comment:
classifier 'dist' or the artifact name will be `atlasdb-timelock-server-version.sls.tgz` instead of `atlasdb-timelock-server-version-dist.sls.tgz`